packages/ui/src/ButtonGroup/index.tsx

```diff
@@ -48,6 +50,12 @@ export interface ButtonGroupProps<IconOnly extends boolean> {
    * Will be used for all buttons in the group.
    */
   iconOnly?: IconOnly;
+  /**
+   * In some cases, you will want to disable responsiveness and enforce that
+   * buttons are always rendered in a column, rather than a row (such as in a
+   * narrow layout width, such as a dialog). In those cases, set this to `true`.
+   */
+  column?: boolean;
```

Comment:
Added this for the purpose of the `Dialog` component, which will always display buttons in a column.